rfs/root_file.go

```diff
@@ -66,3 +66,12 @@ func (f *rfsFile) Write(ctx context.Context, data []byte, off int64) (written ui
 
 	return f.loopbackFile.Write(ctx, data, off)
 }
+
+func (f *rfsFile) Lseek(ctx context.Context, off uint64, whence uint32) (uint64, syscall.Errno) {
```

Comment:
was this just for debugging? Can we get rid of it now?

Reply:
I originally added this for debugging purposes but I'd like to leave it here. I couldn't manage to get this function invoked but I   just didn't try hard enough; it's definitely being used out in the wild.

Reply:
However if you insist I can get rid of it.